src/Counter.php

```diff
@@ -75,7 +75,7 @@ public function hit(string $id): CounterState
         do {
             // Last increment time.
             // In GCRA it's known as arrival time.
-            $lastIncrementTimeInMilliseconds = $this->timer->nowInMilliseconds();
+            $lastIncrementTimeInMilliseconds = round((float)$this->timer->now()->format('U.u') * 1000);
```

Comment:
```suggestion
            $lastIncrementTimeInMilliseconds = round($this->timer->now()->format('U.u') * 1000);
```

Reply:
That would make static analysis complain.